packages/server/test/HeadHandler.test.ts

```diff
@@ -67,11 +65,11 @@ describe('HeadHandler', () => {
     const file = new Upload({
       id: '1234',
       offset: 0,
-      metadata: 'filename d29ybGRfZG9taW5hdGlvbl9wbGFuLnBkZg==,is_confidential',
```

Comment:
Is there another test covering the encoding and decoding of metadata? Because this one does not cover this aspect after your changes.

Reply:
Yes, there is another test in this file:

```js
  it('should resolve with metadata', async () => {
    const file = new Upload({
      id: '1234',
      offset: 0,
      metadata: {is_confidential: undefined, foo: 'bar'},
    })
    fake_store.getUpload.resolves(file)
    await handler.send(req, res)
    assert.equal(res.getHeader('Upload-Metadata'), 'is_confidential,foo YmFy')
  })

```